eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase.php

```diff
@@ -2058,6 +2058,54 @@ public function copyRelations($originalContentId, $copiedContentId, $versionNo =
         $stmt->execute();
     }
 
+    /**
+     * Updates Content's attribute text value.
+     *
+     * @param int $attributeId
+     * @param int $version
+     * @param string $text
+     */
+    public function updateContentObjectAttributeText($attributeId, $version, $text)
+    {
+        $query = $this->connection->createQueryBuilder();
+        $query
+            ->update('ezcontentobject_attribute', 'oa')
```

Comment:
I would extract the name of the table `ezcontentobject_attribute` and all its occurrences to a constant.

Reply:
The whole file uses hardcoded value - I was not sure that I should change it (or if it is time and place for that). WDYT @alongosz

Reply:
There is a lack of convention also in the docblocks (return vs returns) so clean up is needed anyway. However, indeed let @alongosz decide.